server/src/main/java/com/defold/extender/builders/CSharpBuilder.java

```diff
@@ -177,35 +189,56 @@ private static Path getNativePath(String platform) throws IOException {
         return Paths.get(NUGET_PACKAGES, String.format("microsoft.netcore.app.runtime.nativeaot.%s/%s/runtimes/%s/native", csplatform, dotnetVersion, csplatform));
     }
 
+    private static ArrayList<String> makePathsAbsolute(String basePath, ArrayList<String> files) {
```

Comment:
Currently still needed to have absolute paths for the libraries on macos, as they ship both static and dynamic libs in the same folder (and clang defaults to choosing the dynamic ones)